docs/examples/pipeline/query_pipeline_async.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    # add links from input to retriever (id'ed by chunk_size)
</pre><p>nit: we aren't linking to a retriever in this case, but to a query engine? Did you mean to have it be retrievers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
good catch. doing query engine since easier to measure latency speedup